manifests/source.pp

```diff
@@ -23,7 +23,8 @@
 #   Specifies a distribution of the Apt repository.
 #
 # @param repos
-#   Specifies a component of the Apt repository.
+#   Specifies a component of the Apt repository, Valid options: main, contrib, non-free, universe, multiverse and restricted.
+#   Default 'main'.
```

Comment:
don't mention the default value here. puppet-strings will read it from the source code and add it to the REFERENCE.md automatically.

Reply:
ahh nice, wasn't aware about it. Thanks

Reply:
Tried removed from here and generated REF file and observed it doesn't. So I think it make sense to keep it here?

Reply:
You can see that it generates the default value here: https://github.com/puppetlabs/puppetlabs-apt/blob/978c3431a3ccab8bdd6d55d3bcdd6f22cecacdf0/REFERENCE.md?plain=1#L986

Reply:
Have updated the approach with regex as suggested by @smortex, can you please re-review?
Thanks